data/transactions/logic/assembler.go

```diff
@@ -1938,6 +1938,7 @@ type disassembleState struct {
 	numericTargets bool
 	labelCount     int
 	pendingLabels  map[int]string
+	rerun          bool
```

Comment:
please add a comment for `rerun`, it is not obvious at all

Reply:
comment added